doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -15785,6 +15787,18 @@ The name of the supply fan outlet node.
 
 The name of a NodeList object containing the names of the HVAC system nodes or the HVAC System Node Name where temperature setpoints will be established by this setpoint manager.
 
+#### Field: Cooling Coil Inlet Node Name
+
+The name of the cooling coil inlet node.
+
+#### Field: Cooling Coil Outlet Node Name
+
+The name of the cooling coil outlet node.
+
+#### Field: Minimum Temperature at Cooling Coil Outlet Node
+
+In order to prevent cooling coil freezing, the minimum temperature at the cooling coil outlet is required based on requirement of ASHRAE Standard 90.1. The default value is 7.2C.
```

Comment:
Done. Thanks.
@mjwitte I have a thought. When I search "economizer" over IO Ref, I can find words and field name containing economizer. However, I could not find any object name using Economizer. I assume two objects (Controller:OutdoorAir and OutdoorAir:Mixer) represent an economizer. Do you think we may need an object to represent an HVAC system component "Economizer"?